### PR TITLE
Update to sbt-lucuma 0.6

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -68,6 +58,4 @@ val lucumaSchemas =
     )
     .defaultAxes(VirtualAxis.jvm, VirtualAxis.scalaPartialVersion(scala2Version))
     .jvmPlatform(allVersions)
-    .jsPlatform(allVersions,
```
Comment (cquiroz) [inline build.sbt:71]: Isn't this required for tests?

Comment (armanbilge) [reply build.sbt:71]: Nope. Legacy setting.

Comment (cquiroz) [reply build.sbt:71]: So munit doesn't need it anymore?

Comment (armanbilge) [reply build.sbt:71]: Yep, as of this PR.
https://github.com/scalameta/munit/pull/376

**Review by cquiroz — APPROVED** [review]
LGTM, thanks

**Pull request merged**